pages/faq.tsx

```diff
@@ -23,7 +23,7 @@ const questions: ReadonlyArray<{
   {
     question: `How long will the Phase 2 incentivized testnet run for?`,
     id: `testnet-duration`,
-    answer: `The testnet will run from May 10, 2022 @ 12:00 PDT until the Iron Fish node is stable, feature-complete, and ready for mainnet 🎉`,
+    answer: `Phase 2 will run from May 10, 2022 @ 12:00 PDT until November 15, 2022. The testnet will run until the Iron Fish node is stable, feature-complete, and ready for mainnet 🎉`,
```

Comment:
Wording is a little clunky. Could change to "The testnet will continue to run..."

Reply:
It's fine. Can we put the time as well? Noon PDT?

Reply:
Done